app/assets/stylesheets/base.scss

```diff
@@ -0,0 +1,47 @@
+main {
+
+  h1 {
+    @extend .heading-xlarge !optional;
```

Comment:
This bug report might be related: https://github.com/sass/sass/issues/2177
I'll chase that up when I'm back from my holiday.